### ISSUE TITLE
init container missing from ES catalog crd

### ISSUE THREAD

**Issue by tamalsaha** [kubedb/project#450]
https://github.com/kubedb/elasticsearch/blob/50374386935fdcf6111dc31ac8f7f1bbba2a4100/pkg/controller/statefulset.go#L79

**Comment by tamalsaha** [comment]
![init-container-catalog](https://user-images.githubusercontent.com/94814/54017716-b8c05880-413b-11e9-8af1-e2c4b5439165.png)

**Comment by tamalsaha** [comment]
```
	// Init container Image
	InitContainer ElasticsearchVersionInitContainer `json:"initContainer"`
```

**Comment by tamalsaha** [comment]
Fixed in 0.11.0

**Comment by tamalsaha** [comment]
xref: https://github.com/kubedb/mongodb/pull/149
